sound/soc/soc-topology.c

```diff
@@ -2612,7 +2612,7 @@ static int soc_tplg_manifest_load(struct soc_tplg *tplg,
 		ret = tplg->ops->manifest(tplg->comp, tplg->index, _manifest);
 
 	if (!abi_match)	/* free the duplicated one */
-		kfree(_manifest);
+		devm_kfree(tplg->dev, _manifest);
```

Comment:
See my comments to the next commit, this conversion doesn't seem to make sense to me. Here a temporary buffer is freed after being allocated a few lines above, nothing to device-manage